pkgs/development/python-modules/evosax/default.nix

```diff
@@ -0,0 +1,88 @@
+{
+  lib,
+  buildPythonPackage,
+  pythonOlder,
+  fetchFromGitHub,
+
+  # build-system
+  setuptools,
+  wheel,
+
+  # dependencies
+  chex,
+  dotmap,
+  flax,
+  jax,
+  jaxlib,
+  matplotlib,
+  numpy,
+  pyyaml,
+
+  # checks
+  # brax, (unpackaged)
+  # gymnax, (unpackaged)
+  pytestCheckHook,
+  torch,
+  torchvision,
+}:
+
+buildPythonPackage rec {
+  pname = "evosax";
+  version = "0.1.6";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "RobertTLange";
+    repo = "evosax";
+    rev = "refs/tags/v.${version}";
+    hash = "sha256-v8wRiWZlJPF9pIXocQ6/caHl1W4QBNjkmuImJ6MAueo=";
+  };
+
+  build-system = [
+    setuptools
+    wheel
+  ];
+
+  dependencies = [
+    chex
+    dotmap
+    flax
+    jax
+    jaxlib
+    matplotlib
+    numpy
+    pyyaml
+  ];
+
+  pythonImportsCheck = [ "evosax" ];
+
+  preCheck = ''
+    export HOME=$(mktemp -d)
+  '';
+
+  nativeCheckInputs = [
+    # brax
+    # gymnax
+    pytestCheckHook
+    torch
+    torchvision
+  ];
+
+  disabledTests = [
+    # Requires unpackaged gymnax
+    "test_env_ffw_rollout"
+
+    # Tries to download a data set from the internet
+    "test_vision_fitness"
+  ];
+
+  meta = {
+    description = "Evolution Strategies in JAX";
+    homepage = "https://github.com/RobertTLange/evosax";
+    changelog = "https://github.com/RobertTLange/evosax/blob/${src.rev}/CHANGELOG.md";
+    license = lib.licenses.asl20;
```

Comment:
We may need to ask upstream if it is an Apache or an MIT license.
https://github.com/RobertTLange/evosax/blob/v.0.1.6/CONTRIBUTING.md#any-contributions-you-make-will-be-under-the-mit-software-license
https://github.com/RobertTLange/evosax/blob/3c4f34828094e2b1e6798003190f10b67e2a4a9e/setup.py#L62

Reply:
I think that this project is licensed under Apache 2.0.
This MIT mention looks like a `setup.py` copy-paste to me...

@RobertTLange, could you please confirm what is the license of `evosax` ?

Reply:
@natsukium should we keep this PR blocked because of this license issue ?
Personally, I would trust the main `LICENSE` file from the repo and move on. I have opened an issue upstream to ask for clarification.
What do you think ?

Reply:
Let's go ahead. We can update it later.